### PR TITLE
nft market 

### PR DESCRIPTION
**Pull request by abrzezinski94**

### CODE REVIEW

**Comment by tlrsssss** [comment]
code seems fine. i'm not very familiar with this feature or NFTs in general so should have @mschneider review for UX

File: components/nftMarket/AllBidsView.tsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+  TrBody,
+  TrHead,
+} from '@components/shared/TableElements'
+import { ImgWithLoader } from '@components/ImgWithLoader'
+import NftMarketButton from './NftMarketButton'
+import { abbreviateAddress } from 'utils/formatting'
+import { NoSymbolIcon } from '@heroicons/react/20/solid'
+
+const AllBidsView = () => {
+  const wallet = useWallet()
```
Comment (tlrsssss) [inline components/nftMarket/AllBidsView.tsx:30]: Can you destructure the variables you're using from `useWallet`? It's a bit more concise and prevents having to call `wallet.wallet` when you need to access the actual wallet object.

File: components/nftMarket/ListingsView.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+import { MANGO_MINT_DECIMALS } from 'utils/governance/constants'
+// import { useTranslation } from 'next-i18next'
+// import ResponsivePagination from 'react-responsive-pagination'
+import { ImgWithLoader } from '@components/ImgWithLoader'
+import NftMarketButton from './NftMarketButton'
+
+const filter = [ALL_FILTER, 'My Listings']
+
+const ListingsView = () => {
+  const wallet = useWallet()
```
Comment (tlrsssss) [inline components/nftMarket/ListingsView.tsx:25]: same here

File: hooks/useMetaplex.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { Metaplex, walletAdapterIdentity } from '@metaplex-foundation/js'
+import { useWallet } from '@solana/wallet-adapter-react'
+import mangoStore from '@store/mangoStore'
+import metaplexStore from '@store/metaplexStore'
+import { useEffect } from 'react'
+
+export default function useMetaplex() {
+  const connection = mangoStore((s) => s.connection)
+  const wallet = useWallet()
```
Comment (tlrsssss) [inline hooks/useMetaplex.ts:9]: same

**Review by tlrsssss — CHANGES_REQUESTED** [review]

File: hooks/useMetaplex.ts (hunk 3 above)
Comment (abrzezinski94) [reply hooks/useMetaplex.ts:9]: fixed other places but in here im using whole wallet for metaplex identity

**Review by tlrsssss — APPROVED** [review]

**Pull request merged**